lib/cron_parser.rb

```diff
@@ -80,18 +80,17 @@ def interpret_vixieisms(spec)
   # returns the next occurence after the given date
   def next(now = @time_source.now, num = 1)
     t = InternalTime.new(now, @time_source)
-
-    unless time_specs[:month][0].include?(t.month)
+    if !time_specs[:month][0].include?(t.month)
```

Comment:
Sorry about that! I actually found that to be more readable while I was trying to figure out the operation of things, but forgot to revert the change. Will certainly address.